client/web/src/features/series/actionTypes.ts

```diff
@@ -0,0 +1,4 @@
+export const SET_CURRENT_SERIE = 'SET_CURRENT_SERIE'
```

Comment:
```suggestion
export const SET_CURRENT_SERIE = 'SET_CURRENT_SERIES'
```
'Serie' doesn't exist in English